components/binding/README.md

```diff
@@ -20,13 +19,13 @@ helm install binding ./charts/binding --wait
 
 Go through the short tutorial that presents the useage and capabilities of the component.
 
-1. Register a Target Kind. It's a resource that defines how and where environments can be injected.
+1. Register a TargetKind. It's a resource that defines how and where environments can be injected.
```

Comment:
Are you sure TargetKind should be in camelcase?

Reply:
I'm sure about nothing here :D
It's called "a resource". And it was one of the changes from https://github.com/kyma-project/kyma/pull/13634/files which I tried to revert.